models/reviews.model.js

```diff
@@ -0,0 +1,15 @@
+const db = require('../db/connection')
+
+exports.fetchReviewByID = (id) => {
+    if(!id){
+        return Promise.reject({status: 400, msg: "Invalid Id"})
+    }
+    return db.query(`SELECT * FROM reviews WHERE review_id = ${id}`)
```

Comment:
This could open yourself up to SQL injection. We can avoid this using the $1 syntax. Please change this line to ensure it is safe from SQL injection